### PR TITLE
Arrays

### PR DESCRIPTION
**Pull request by ritz-gupta**
These are a few programs based on array. 

Would be delightful if someone checked these and suggested a better way which makes them efficient.

### CODE REVIEW

**Comment by ritz-gupta** [comment]
@GetPsyched @Maan2003 
> Come on, do something.

File: 12 add subtract matrices.cpp [hunk 1]
```diff
@@ -30,7 +33,7 @@ main()
 	if((row_1==row_2)&&(col_1==col_2)){
 		printf("\nThe matrices are compatible for addition and subtraction");
 		printf("\nTo add: Press 1\nTo subtract: Press 2\n");
-		int a;																			//To know what we want to do with the matrices
+		int a;												//To know what we want to do with the matrices
```
Comment (maan2003) [inline 12 add subtract matrices.cpp]: Comment is too far in the right. Most common practice is to place it on separate line.
Do this for other comments like this as well
```suggestion
 		// To know what we want to do with the matrices
		int a;
```

File: 9 max marks.cpp [hunk 2]
```diff
@@ -1,5 +1,9 @@
+/*The marks of students are entered in Physcis,Chemistry,Maths. We need to finnd the sum total of marks of the student who scored mamximum marks in any of the subjects.
+  Also,report if the same student has maximum marks in two or more subjects */
```
Comment (maan2003) [inline 9 max marks.cpp]: Consider using single line comments and limited the size of line
```suggestion
// The marks of students are entered in Physics, Chemistry, Maths.
// We need to find the sum total of marks of the student who scored maximum marks in any of the subjects.
// Also, report if the same student has maximum marks in two or more subjects
```

**Review by maan2003 — COMMENTED** [review]
Looks great! It would be nice you used `.c` files instead.

**Comment by maan2003** [comment]
Press the green button!

**Comment by ritz-gupta** [comment]
> Press the green button!

Already did

**Comment by maan2003** [comment]
Github times everything :smile: image

**Pull request merged**